lib/protocol/moveTo.js

```diff
@@ -67,5 +67,18 @@ export default function moveTo (element, xoffset, yoffset) {
         }, { ELEMENT: element }, target.x, target.y)
     }
 
-    return this.requestHandler.create('/session/:sessionId/moveto', data)
+    return this.requestHandler.create('/session/:sessionId/moveto', data).catch(err => {
+        return findMoveToCoordinates.call(this, element, xoffset, yoffset).then((coordinates) => {
```

Comment:
I'd feel better if you didn't make this run on every single failed moveTo() (especially since this chains into the actions api, that most browsers don't support).

Reply:
Updated!